cli/src/main/java/com/box/l10n/mojito/cli/command/CommandHelper.java

```diff
@@ -51,6 +60,10 @@ public class CommandHelper {
   /** logger */
   static Logger logger = LoggerFactory.getLogger(CommandHelper.class);
 
+  private static final String TIMEFRAMES_REGEX = "^([1-9][0-9]{0,2}[%s])+$";
```

Comment:
Could this name be suffixed with _PLACEHOLDER, as it's not obvious that %s isn't the format that the regex is looking for - and that part isn't actually part of the regex pattern and will be replaced at runtime by something else.

It might be beneficial to also add a short comment explaining that behaviour to any future readers of the code.